src/resources/views/crud/list.blade.php

```diff
@@ -145,6 +145,15 @@
   <link rel="stylesheet" href="{{ asset('packages/backpack/crud/css/form.css') }}">
   <link rel="stylesheet" href="{{ asset('packages/backpack/crud/css/list.css') }}">
 
+  @if ($crud->getOperationSetting('hideTableButtonsText') ?? config('backpack.crud.operations.list.hideTableButtonsText', false))
+  <style>
+    @media screen and (min-width: 1025px) {
```

Comment:
I wrote about it in my previous comment. https://github.com/Laravel-Backpack/CRUD/pull/3320#issuecomment-742478423

Reply:
@pxpm I think it's the best way to do this 👌
I though about that somewhere, I was doubting if we should go "so far" in terms of customization. 

But yes, I think we should 😅

Reply:
ping @tabacitu related to https://github.com/Laravel-Backpack/CRUD/pull/3320#issuecomment-742478423

Let us know what you think :)

Reply:
Another question @tabacitu, I don't particularly like to pollute the `list.blade.php` with these inline CSS rules.
I think the proper way to do this would be to add the classes to the table, `hideBtnTextMobile` and/or `hideBtnTextDesktop`, and then, add the css to the `list.css`. That's the correct approach I think. 

The downside, is that this PR will only be available for new installs of v4.1, only when publishing the assets.

Tell me what you think, @pxpm can you give us your opinion on that too?